### PR TITLE
Enable confirm error button in Husqvarna Automower by default

### PR DESCRIPTION
**Pull request by Thomas55555**
## Proposed change

Enable confirm error button in Husqvarna Automower by default as the API now provides the capability of the device to do that.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/123927
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Simplified the description of the button entity in the Husqvarna Automower integration, enhancing clarity regarding its functionality.

### CODE REVIEW

File: source/_integrations/husqvarna_automower.markdown [hunk 1]
```diff
@@ -107,7 +107,7 @@ The integration will create the following binary sensors:
 
 ### Button (if available)
 
-The integration will create a button entity for confirming minor mower errors. This entity is disabled by default. You have to enable it manually. The API can't detect if the mower has the capability to confirm minor errors remotely. Before enabling this function, refer to the mower documentation.
+The integration will create a button entity for confirming minor mower errors.
```
Comment (gjohansson-ST) [inline source/_integrations/husqvarna_automower.markdown:110]: Should we clarify here that the button only exist if supported by the mower?

**Review by gjohansson-ST — CHANGES_REQUESTED** [review]

File: source/_integrations/husqvarna_automower.markdown (hunk 1 above)
Comment (Thomas55555) [reply source/_integrations/husqvarna_automower.markdown:110]: It's already mentioned in the title of the button entity. I think, that's enough.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Thomas55555 :+1:

File: source/_integrations/husqvarna_automower.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/husqvarna_automower.markdown:110]: @gjohansson-ST, are you good with the note in the title?

Comment (gjohansson-ST) [reply source/_integrations/husqvarna_automower.markdown:110]: Didn't see that. I'm good 👍

**Review by gjohansson-ST — APPROVED** [review]

**Pull request merged**